### ISSUE TITLE
Research APIs to interact with Apache Pulsar (type script)

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#8]
## User story
1. As a PO
2. I need to know if/how it is possible to interact with an Apache Pulsar Instance from the Browser
3. So that I understand how to move on with the project

Link to [knowledge base](https://docs.google.com/document/d/1GjkRqE_nvhg18v1hVktjGBvlwoXAfjvcbwZEnBUc7GI/edit?usp=sharing)

## Acceptance criteria
* Markdown document of following information:
    * [ ] Description of how to interact with Apache Pulsar (Interfaces)
    * [ ] What libraries exist to facilitate the interaction
    * [ ] Is some form of identification (API Keys, Password, ...) necessary
    * [ ] Identification of possible pitfalls
* (Optional but desirable!: try to test API with short demo script)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by jnsrnhld** [comment]
Linked my draft doc to this issue (https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/pull/15). Feel free to extend it.
